docs/howto/features/per-user-db.md

```diff
@@ -0,0 +1,130 @@
+# Setup a database server per user
+
+We can support providing a single, isolated database *server* per user. This
+is extremely helpful for teaching cases, as users can do whatever they want with
+their server without affecting other users.
+
+## Setting up the backing disk for the database
+
+Each database server needs a backing disk to store its data. We have custom code in
+our `basehub/values.yaml` that creates arbitrary extra disks per user that can be used.
+
+```yaml
+jupyterhub:
+  custom:
+    singleuser:
+      extraPVCs:
+        - name: postgres-{username}
+          class: standard
+          capacity: 1Gi
+```
+
+This will create an additional Kubernetes [Persistent Volume Claim](https://kubernetes.io/docs/concepts/storage/persistent-volumes/)
+per user, named `postgres-<name-of-user>`, created with Kubernetes [Storage
+Class](https://kubernetes.io/docs/concepts/storage/storage-classes/), and capacity of
+`1Gi` (remember it is `1Gi`, not `1G`!).
+
+`capacity` can be increased later on if needed and kubernetes will
+[automatically resize](https://kubernetes.io/blog/2018/07/12/resizing-persistent-volumes-using-kubernetes/) the disk
+when users start / stop the server. However, the storage class *can not* be changed
+afterwards - so users will have to decide if they want spinning disks (slower, chepeaper)
+or ssd disks (faster, much more expensive) at the start.
```

Comment:
Where can the different `class` options can be found? I assume `standard` is for the hdd right?

Reply:
Yes! @GeorgianaElena I added both the command used to describe the options, as well as the available options for GKE and AWS now.

Reply:
🎉 Thank youuuuu!